src/components/EpisodesList.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import axios from 'axios';
+import EpisodeCard from './EpisodeCard';
+import styled from 'styled-components';
+
+const CardsBox = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  align-items: stretch;
+  justify-items: center;
+`;
+
+const StyledForm = styled.form`
+  width: 35%;
+  margin: 1.5rem auto;
+  font-family: 'Gaegu', cursive;
+  font-size: 2rem;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  align-items: center;
+  input {
+    height: 2rem;
+  }
+`;
+
+const ButtonContainer = styled.div`
+  width: 50%;
+  margin 1.5rem auto;
+  font-family: 'Gaegu', cursive;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: center;
+  button {
+    padding: 0.5rem;
+    margin: 0.25rem;
+    font-family: 'Gaegu', cursive;
+  }
+`;
+
+export default function LocationsList() {
+    const [searchTermEpisode, setSearchTermEpisode] = useState('');
+    const [searchResultsEpisode, setSearchResultsEpisode] = useState([]);
+    const [page, setPage] = useState(`page=1`);
+
+    function getPage(direction) {
```

Comment:
Why did you decide to write some functions multiple times? -- Also chatted a little bit about this and you could pass the general function from app or a different parent component.